control-announcements/src/main/java/org/oskari/announcements/helpers/Announcement.java

```diff
@@ -0,0 +1,69 @@
+package org.oskari.announcements.helpers;
+
+public class Announcement {
+    private Integer id;
+    private String title;
+    private String content;
+    private String begin_date;
+    private String end_date;
```

Comment:
With the Mybatis mapper taking dates as param shouldn't these be dates as well?

Reply:
Those are strings everywhere and then cast as date, unless I'm missing something (?)